rosbag2/include/rosbag2/writer.hpp

```diff
@@ -95,13 +95,19 @@ class ROSBAG2_PUBLIC Writer
    */
   virtual void write(std::shared_ptr<SerializedBagMessage> message);
 
+  /**
+   * Check if the current recording database file needs to be split and rolled over to new file.
+   */
+  virtual bool should_split_database() const;
```

Comment:
should this be named something more generic - e.g. `should_split_bagfile`? since database is an implementation detail of one storage implementation

Reply:
Renamed to `should_split_bagfile`